test/extract_abi.sh

```diff
@@ -0,0 +1,14 @@
+# Extract typescript declarations from ABIs JSON files
```

Comment:
Shall we put this script outside the test folder, if it will be an official way to generate the files?
Also, maybe it would be good to copy the instructions how to use the script from the PR to the readme.md file?

Reply:
Yes definitely. I updated the branch, putting the .sh into a newly created scripts folder. I am not sure what is the target setup for a project using abiwan, so I guess that could evolve.
I also updated the readme.